ui/apps/platform/src/Containers/Vulnerabilities/components/SnoozeCvesModal/SnoozeCvesModal.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { Alert, Modal, Text, Button, Flex, Form, Radio, FormGroup } from '@patternfly/react-core';
+import { FormikHelpers, useFormik } from 'formik';
+
+import { resourceTypes } from 'constants/entityTypes';
+import { durations, snoozeDurations } from 'constants/timeWindows';
+import useRestMutation from 'hooks/useRestMutation';
+import { suppressVulns, unsuppressVulns } from 'services/VulnerabilitiesService';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+import { ValueOf } from 'utils/type.utils';
+
+const durationOptions = ['DAY', 'WEEK', 'MONTH', 'UNSET'] as const;
+
+type FormValues = {
+    cves: string[];
+    duration: ValueOf<typeof durations>;
+};
+
+export type SnoozeCvesModalProps = {
+    action: 'SNOOZE' | 'UNSNOOZE';
```

Comment:
Could we utilize the `SnoozeAction` you created in `useSnoozeCVEModal` for this too?